index.html

```diff
@@ -0,0 +1,176 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width,initial-scale=1.0">
+        <title>Website Design Project</title>
+        <link rel="stylesheet" href="style.css" type="text/css">
+    </head>
+    <body>
+        <header class="imagecontainer">
+            <section class="img defineimage"></section>
+            <section class="img defineimage1"></section>
+        </header>
+        <section class="textcontainer">
+            <h1 class="text text1">Rajendra Medini</h1>
+            <h1 class="text text3">Projects</h1>
+            <p class="text text2">Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works. If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
+        </section>
+        <a href="#" class="border-button">Start collaboration</a>
+        <section class="secondimagecontain"></section>
+        
+        <section class="projectpageimage1"></section>
```

Comment:
Please as earlier indicated you can use div tags here instead of sections for nice semantic code
```suggestion
        <div class="secondimagecontain"></div>
        
        <div class="projectpageimage1"></div>
```

Reply:
I have replaced section tags with div tags. Thank you very much for your valuable advice.